src/celengine/console.h

```diff
@@ -42,14 +42,16 @@ class ConsoleStreamBuf : public std::streambuf
     unsigned int decodeShift{ 0 };
 };
 
-
+class Renderer;
 class Console : public std::ostream
 {
  public:
+    EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
does not look like it contains Eigen structs here?

Reply:
I has: `Eigen::Matrix4f mpv;` private member.